packages/babel-core/src/gensync-utils/fs.ts

```diff
@@ -10,12 +10,7 @@ export const readFile = gensync<(filepath: string, encoding: "utf8") => string>(
 
 export const exists = gensync<(filepath: string) => boolean>({
   sync(path) {
-    try {
-      fs.accessSync(path);
-      return true;
-    } catch {
-      return false;
-    }
+    return fs.existsSync(path);
   },
   errback: (path, cb) => fs.access(path, undefined, err => cb(null, !err)),
```

Comment:
I prefer to align the behavior in the sync and async cases.

Since `fs.existsSync` is fast anyway, we can just remove this whole gensync wrapper and directly use `fs.existsSync` in the `findConfigUpwards` function.